tools/dep_updaters/update-icu.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+set -e
+# Shell script to update icu in the source tree to a specific version
+
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
+DEPS_DIR="$BASE_DIR/deps"
+TOOLS_DIR="$BASE_DIR/tools"
+
+[ -z "$NODE" ] && NODE="$BASE_DIR/out/Release/node"
+[ -x "$NODE" ] || NODE=$(command -v node)
+
+NEW_VERSION="$("$NODE" --input-type=module <<'EOF'
+const res = await fetch('https://api.github.com/repos/unicode-org/icu/releases/latest');
+if (!res.ok) throw new Error(`FetchError: ${res.status} ${res.statusText}`, { cause: res });
+const { tag_name } = await res.json();
+console.log(tag_name.replace('release-', ''));
+EOF
+)"
+
+ICU_VERSION_H="$DEPS_DIR/icu-small/source/common/unicode/uvernum.h"
+CURRENT_MAJOR_VERSION=$(grep "#define U_ICU_VERSION_MAJOR_NUM" "$ICU_VERSION_H" | cut -d ' ' -f3 | tr -d '\r')
+CURRENT_MINOR_VERSION=$(grep "#define U_ICU_VERSION_MINOR_NUM" "$ICU_VERSION_H" | cut -d ' ' -f3 | tr -d '\r')
+
+CURRENT_VERSION="$CURRENT_MAJOR_VERSION-$CURRENT_MINOR_VERSION"
+
+echo "Comparing $NEW_VERSION with $CURRENT_VERSION"
+
+if [ "$NEW_VERSION" = "$CURRENT_VERSION" ]; then
+  echo "Skipped because icu is on the latest version."
+  exit 0
+fi
+
+NEW_MAJOR_VERSION=$(echo "$NEW_VERSION" | cut -d '-' -f1)
+NEW_MINOR_VERSION=$(echo "$NEW_VERSION" | cut -d '-' -f2)
```

Comment:
sorry I'm not sure I understood the suggestion

Reply:
If ICU releases a `73.1.1` version this code won't work, it will be `release-73-1-1`. Instead of working with the pieces just take the whole number:  `73-1` and `73.1` and process it as a whole.

Reply:
I've noticed that release follow the `73-1` structure, there are no `73-1-1` releases in the whole history